CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+v1.5.0
+
+* Added support for operator objects with only one value, eg `a: $ne: $b`
+* Added status code to thrown errors from the request backend.
```

Comment:
https://github.com/resin-io/pinejs-client-js/pull/11 - I was just adding the changelog entry